Pn 4806 fe controlli accessibilita front end pa

NicolaGiornetta opened the pull request:
## Short description
Increased accessibility for PA.

## List of changes proposed in this pull request
- Modified ItemsTable
- Modified several components

## How to test
Check this page: https://pagopa.atlassian.net/wiki/spaces/PN/pages/655557012/LRR+-+Engineering+standards+-+Frontend
There are tools and guidelines about accessibility

ZorbaDimatteo commented on packages/pn-commons/src/components/CustomTooltip.tsx:
```diff
@@ -31,7 +31,7 @@ function CustomTooltip({ openOnClick, tooltipContent, children, sx, onOpen }: Pr
         <Tooltip
           arrow
-          leaveTouchDelay={0} // this makes the tooltip not disappearing after some seconds by click (useful for mobile view)
+          leaveTouchDelay={5000}
```
@NicolaGiornetta why 5000? how does the tooltip behaves in mobile view?

NicolaGiornetta replied:
Because of "dischargeable" tooltip required by accessibility. In mobile, the tooltip will disappear after 5000 seconds. I did not found anything about timing in accessibility, so I set as 5 seconds.

ZorbaDimatteo commented on packages/pn-commons/src/components/Data/ItemsTable.tsx:
```diff
@@ -25,13 +25,16 @@ type Props<ColumnId> = {
   rows,
   sort,
   onChangeSorting,
+  title,
```
@NicolaGiornetta rename title to ariaTitle

ZorbaDimatteo reviewed (changes requested):
@NicolaGiornetta 
I refactored CustomH3 component to SectionHeading component. I should work similar to CustomH3 component, but I used makeStyle hook. Can you tell me if it's ok?

I encountered two issues:
- In recipients section, I cannotselect by keyboard "Aggiungi domicilio digitale" and "Aggiungi domicilio fisico"
- in recipients section, If I add more than one recipient, I cannot select by keyboard the trash can icon to delete one of the recipient

NicolaGiornetta commented:
> @NicolaGiornetta I refactored CustomH3 component to SectionHeading component. I should work similar to CustomH3 component, but I used makeStyle hook. Can you tell me if it's ok?
> 
> I encountered two issues:
> 
> * In recipients section, I cannotselect by keyboard "Aggiungi domicilio digitale" and "Aggiungi domicilio fisico"
> * in recipients section, If I add more than one recipient, I cannot select by keyboard the trash can icon to delete one of the recipient

There are rules about use of spacebar and enter keys. Enter shouldn't be used for checkboxes and radios but spacebar yes.
Reference: https://www.clairecodes.com/blog/2019-11-06-the-roles-of-spacebar-and-enter-in-keyboard-navigation/

In the meanwhile I've fixed the delete button.
